wp/resources/sample_binaries/pointer_flag/main_1.c

```diff
@@ -0,0 +1,15 @@
+int f(){
+    return 1;
+}
+// Type your code here, or load an example.
+int deref(int *num1) {
+    int num2 = f();
```

Comment:
you could have `int num2 = 12;` and then add a `int local = 42;` right after, that way you need to prove that `*num1` doesn't point to the local stack.

This is the main problem we're having anyway.